### PR TITLE
METRON-1541: Mvn clean results in git status having deleted files.

### PR DESCRIPTION
**Pull request by justinleet**
## Contributor Comments
Run
`mvn clean`

then run
`git status`

Status should be clean with no deleted files.

This matches what's used by the other template files.

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```

- [ ] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

**Comment by mmiklavc** [comment]
Should profiler properties also be included? https://github.com/apache/metron/blob/2b4f0b84062d65f9400421d66ec3b7d6d093bebf/metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/templates/profiler.properties.j2

**Comment by mmiklavc** [comment]
And while you're in there, we can probably drop the old enrichment.properties.j2 file from the gitignore

**Comment by justinleet** [comment]
@mmiklavc I don't believe profiler.j2 should. That template looks like it actually gets used directly.  It should probably be made more consistent with the other properties files, but I'm inclined to leave that as follow-on.

If we're dropping the old enrichment.properties.j2, should that be dropped from metron-enrichment?  I'm not familiar enough with how that all lines up to know if they can be safely dropped.

**Comment by mmiklavc** [comment]
@justinleet ok, didn't realize that about the profiler props. I'm pretty sure enrichment.properties.j2 is no longer used as of METRON-1499 - https://github.com/apache/metron/pull/984/files#diff-2e9198cfca415aeee9f9118ab279aec0

**Comment by justinleet** [comment]
@nickwallen Are we good to kill the old enrichment.properties.j2 files?

**Comment by mmiklavc** [comment]
Tagging @cestella who was involved with writing the new unified topology.

File: metron-deployment/packaging/ambari/.gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ archive.zip
 elasticsearch.properties.j2
 hdfs.properties.j2
 enrichment.properties.j2
```
Comment (cestella) [inline metron-deployment/packaging/ambari/.gitignore]: I don't think we need this anymore, right?

**Comment by nickwallen** [comment]
> @justinleet  Are we good to kill the old enrichment.properties.j2 files?

Yes, I should have deleted that file in #984 .  

Thanks @justinleet for the clean-up on this.  The enrichment-*.properties.j2 live in metron-enrichment/ and are copied into the mpack during the build.  

+1

**Comment by nickwallen** [comment]
Sorry for the delay.  My email filters are clearly filtering out my Github notifications.

**Pull request closed without merging**